extensions/replication/queueProcessor/task.js

```diff
@@ -35,7 +35,6 @@ function initAndStart() {
             setTimeout(initAndStart, 5000);
             return;
         }
-        console.log('management init done');
         queueProcessor.start();
         // setInterval(initManagement, 5000, error => {
```

Comment:
Since moving the `setInterval` to `initManagement` maybe we can go ahead and remove this comment?